bats/tests/pyrsia_node_docker_build_service.bats

```diff
@@ -109,7 +110,8 @@ setup() {
 
   # try to authorize again
   run "$PYRSIA_CLI" authorize --peer "$PEER_ID"
-  assert_output "Authorize request failed with error: HTTP status client error (400 Bad Request) for url (http://${CLIENT_HOSTNAME}/authorized_node)"
+  assert_output --partial "Authorize request failed with error: HTTP status error (400 Bad Request) for url (http://${CLIENT_HOSTNAME}/authorized_node)"
```

Comment:
@ihcomega56 I don't think this check has to contain a very long string since it’s “--partial”, one or two words (e.g. '400 Bad' or similar) should be sufficient and also less prone to failure in case of the CLI related changes.

Reply:
@karolh2000 Thank you. Shortened other strings as well.